spec/features/resgistration_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.feature 'User registration', type: :feature, ui: true do
+  given!(:contest) { create :contest }
+
+  def registration_url
+    "/contests/#{contest.id}/users/new"
+  end
```

Comment:
Consider using `given`:

```suggestion
  given(:registration_url) { "/contests/#{contest.id}/users/new" }
```

Also, `registration_path` seems like a better name.

Reply:
fixed